### ISSUE TITLE
AttributeError: 'PDFStream' object has no attribute 'replace'

### ISSUE THREAD

**Issue by panoptikum** [pdfminer/pdfminer.six#210]
Hello everybody,

At the moment I'm parsing tons of PDFs, but pdfminer.six fails on one of them. Any suggestions? I can open the PDF, but maybe pdfminer.six can't handle it properly. All the other PDFs origin from the same author/organization...

```
Traceback (most recent call last):
  File "/home/felix/anaconda3/bin/pdf2txt.py", line 136, in <module>
    if __name__ == '__main__': sys.exit(main())
  File "/home/felix/anaconda3/bin/pdf2txt.py", line 131, in main
    outfp = extract_text(**vars(A))
  File "/home/felix/anaconda3/bin/pdf2txt.py", line 63, in extract_text
    pdfminer.high_level.extract_text_to_fp(fp, **locals())
  File "/home/felix/anaconda3/lib/python3.6/site-packages/pdfminer/high_level.py", line 82, in extract_text_to_fp
    interpreter.process_page(page)    
  File "/home/felix/anaconda3/lib/python3.6/site-packages/pdfminer/pdfinterp.py", line 852, in process_page
    self.render_contents(page.resources, page.contents, ctm=ctm)
  File "/home/felix/anaconda3/lib/python3.6/site-packages/pdfminer/pdfinterp.py", line 862, in render_contents
    self.init_resources(resources)
  File "/home/felix/anaconda3/lib/python3.6/site-packages/pdfminer/pdfinterp.py", line 362, in init_resources
    self.fontmap[fontid] = self.rsrcmgr.get_font(objid, spec)
  File "/home/felix/anaconda3/lib/python3.6/site-packages/pdfminer/pdfinterp.py", line 212, in get_font
    font = self.get_font(None, subspec)
  File "/home/felix/anaconda3/lib/python3.6/site-packages/pdfminer/pdfinterp.py", line 203, in get_font
    font = PDFCIDFont(self, spec)
  File "/home/felix/anaconda3/lib/python3.6/site-packages/pdfminer/pdffont.py", line 658, in __init__
    self.cmap = CMapDB.get_cmap(name)
  File "/home/felix/anaconda3/lib/python3.6/site-packages/pdfminer/cmapdb.py", line 259, in get_cmap
    data = klass._load_data(name)
  File "/home/felix/anaconda3/lib/python3.6/site-packages/pdfminer/cmapdb.py", line 233, in _load_data
    name = name.replace("\0", "")
AttributeError: 'PDFStream' object has no attribute 'replace'

```

**Comment by panoptikum** [comment]
Yes.

[Problematic PDF](http://starweb.hessen.de/cache/PLPR/18/9/00119.pdf)

OR

[Problematic PDF](http://starweb.hessen.de/cache/PLPR/18/7/00117.pdf)

**Comment by skewed91** [comment]
Same error

**Comment by bimbocant** [comment]
I'm encountering the same error when trying to parse

**Comment by euarthuurr** [comment]
I am encountering the same error when I run on the Mac (OS 10.10.5), but this error doesn't happen when I'm using Windows 10.

**Comment by euarthuurr** [comment]
If I open file Problematic PDF and saved as PDF on Mac, pdfminer works correctly.
[Problematic PDF works](https://drive.google.com/file/d/1JDxNzMcU79qdMHocWFhbme9D5Z38ao_i/view?usp=sharing)

**Comment by bimbocant** [comment]
I tried saving it as PDF on Windows 10. Still not working.

**Comment by bimbocant** [comment]
#228 works fine.

**Comment by pforero** [comment]
Hi! I am getting still the same issue. I have found that on the problematics PDFs the specs seemed to be mixed. The 'Encoding' spec is a PDFStream object, and the 'ToUnicode' spec is 'Identity-H'. I have also tried #228 but it does not resolve it, the final 'name' value is 'CM20' which does not Map to Identity-H/V.

I am not sure why for some PDFs (it is a reasonable amount of them, close to 10% of those I have) it mixes ToUnicode and Encoding. Hope you guys can have some further insight on the cause

**Comment by tangxfei** [comment]
To modify the “cmapdb.py”（\Lib\site-packages\pdfminer\cmapdb.py）
Line:233
    def _load_data(klass, name):
        name = name.replace("\0","")       #Modify to=> name=str(name).replace("\0","")

The problem can be solved.

**Comment by benzkji** [comment]
I am encountering the same error, on (probably) one of around 1000 files. Will investigate, report if re-saving it (I am on Linux) will help...

@tangxfei mind a PR ;-)

**Comment by igormp** [comment]
@benzkji there's already a PR at #228 that fixes this, but the project seems kinda dead

**Comment by vinayak-mehta** [comment]
@0xabu Looks like you have merge access since you merged #230. Please look into #228 and merge it :)

**Comment by vinayak-mehta** [comment]
@0xabu Do you also have access to push a new release to PyPI? cc: @ganeshtata

**Comment by 0xabu** [comment]
@vinayak-mehta Someone (I think @goulu) merged some bugfix PRs and then added me to the org back in early 2017, but like you I just depend on pdfminer so I'm not comfortable (not to mention don't have time) taking on responsibility for it. I don't know anything about the PyPi package.

**Comment by pinnnip** [comment]
I have tried all the methods mentioned in all the related threads, it is still not working on most of my PDFs. Are there any more updates?

**Comment by pinnnip** [comment]
Some of output only has a string '\x0c' with no content from pdf file.

**Comment by pietermarsman** [comment]
Fixed by #283
